src/sections/enterprise-offering/enterprise-bullets/enterprise-bullets.module.scss

```diff
@@ -0,0 +1,13 @@
+.benefits {
+	grid-gap: 40px;
+}
+.icon {
+	margin: 0 auto;
+	height: 50px;
+	width: 50px;
```

Comment:
I think you can just use `width`. the height is not needed because images keep the ratio

Reply:
done